src/classes/AccountHelperClass.cls

```diff
@@ -0,0 +1,49 @@
+public with sharing class AccountHelperClass{
```

Comment:
Change the name of the class it should be AccountTriggerHelper or AccountTriggerHandler